[2025-09-20 22:50:15] Application is DOWN (HTTPConnectionPool(host='localhost', port=4499): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E89CF44BE0>: Failed to establish a new connection: [WinError 10061] No connection could be made because the target machine actively refused it')))
